osu.Game/Screens/Play/BeatmapMetadataDisplay.cs

```diff
@@ -41,18 +43,22 @@ public bool Loading
             }
         }
 
-        public BeatmapMetadataDisplay(WorkingBeatmap beatmap, Bindable<IReadOnlyList<Mod>> mods, Drawable facade)
+        public BeatmapMetadataDisplay(WorkingBeatmap beatmap, Bindable<IReadOnlyList<Mod>> mods, [CanBeNull] Drawable logoFacade)
```

Comment:
is this ctor change for the test? it seems unnecessary, as you could pass in `Empty()` from the test instead of adding it in here.